recipe/meta.yaml

```diff
@@ -11,43 +11,48 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record=files.txt
-  skip: true  # [win or py3k]
+  number: 1
```

Comment:
Since we're bumping the version number, the build number can be reset to 0.

Reply:
oops, thanks for catching that!